### DISCUSSION TITLE
<The api_id/api_hash combination is invalid>

### DISCUSSION THREAD

**Discussion by ko-gif** [community/community#43387] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

telethon.errors.rpcerrorlist.ApiIdInvalidError: The api_id/api_hash combination is invalid (caused by SendCodeRequest)

anyone can help me to fix,,pleasee

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ko-gif ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
